README.md

```diff
@@ -145,52 +144,41 @@ In order to develop your functions locally the following is the recommended prac
 {
   "functions": "./functions/index.ts",
   "vendor": "./vendor",
-  "preVendor": true,
   "schemaMode": "INFER"
 }
 ```
-* (Optionally) If you want your development vendor and inference resources to be used to speed up deployment, add the following to your `./config.json`:
-  ```json
-  {
-    "functions": "./functions/index.ts",
-    "vendor": "./functions/vendor",
-    "preVendor": true,
-    "schemaLocation": "./functions/schema.json",
-    "schemaMode": "INFER"
-  }
-  ```
-  * Make sure to .gitignore your computed `vendor` and `schema.json` files.
+  * Make sure to .gitignore your computed `vendor` files.
 * Start the connector
 ```sh
 deno run -A --watch --check https://deno.land/x/hasura_typescript_connector/mod.ts serve --configuration ./config.json
 ```
-* (Optionally) Add a test-suite to your functions. See [Deno Testing Basics](https://docs.deno.com/runtime/manual/basics/testing).
+* (Optionally) Add a test-suite for your functions. See [Deno Testing Basics](https://docs.deno.com/runtime/manual/basics/testing).
 
 
 ## Config Format
 
 The configuration object has the following properties:
 
 ```
-  functions      (string): Location of your functions entrypoint (default: ./functions/index.ts)
+  functions      (string): Location of your functions entrypoint
```

Comment:
Why no documented default for functions, but present for preVendor, schemaMode?

Reply:
You can't default `functions` any more, you must specify it as something yourself, so there is no default. You will get an error if you leave it out or put in a blank string. In all the places where we control the locations of the functions (ie. in docker) we specify it. We specify it in all the documentation. We also generate a value in a blank configuration object.

Reply:
Is there a reason for this? I thought it was an advantage minimise boilerplate in the config.